wetterdienst/core/scalar/request.py

```diff
@@ -13,13 +12,9 @@
 import numpy as np
 import pandas as pd
 import pytz
-
-with warnings.catch_warnings():
-    warnings.simplefilter("ignore")  # ignore missing python-levenhstein
-    from fuzzywuzzy import fuzz, process
-
 from measurement.measures import Distance
 from measurement.utils import guess
+from rapidfuzz import fuzz, process
```

Comment:
This order appears to make the linter happy, but I have absolutely no idea what criteria the list is sorted by.

Reply:
It is sorted firstly by level (main libraries, installed libraries, local libraries) and secondly by the starting letters... If you run 
poe format

you'll get what we want almost 99%! :-D